### DISCUSSION TITLE
Scaling of site scores from dbRDA for PROTEST / Procrustes analysis

### DISCUSSION THREAD

**Discussion by grahamfrank** [vegandevs/vegan#683] [Q&A] [answered]
I have noticed that I get the same protest() results when I use dbrda objects in the call to protest() directly, or if I input site scores pulled with scores(x, scaling = "none"). However, results differ when I use scores scaled by eigenvalues of each axis, using scores(x, scaling = 'sites'). So, it seems that that the internal scores() call in protest is using unscaled site scores. I also find that the variance of the values from scores(x, scaling = 'sites') is proportional to the eigenvalues of each axis, confirming that those are scaled by the eigenvalues.

Can anyone explain to me why using unscaled site scores is the correct way to go for Procrustes analysis? It seems to me one should prefer the scores to be scaled by the importance of each axis in some way. It is very possible that I am misunderstanding something fundamental about Procrustes analysis, and how it is supposed to be done with constrained ordinations. 

For context, I have community matrices of different taxa at the same set of 70 sites, and am interested in evaluating the correspondence between them when constraining with a set of environmental variables in dbRDA.

Thanks, 
Graham

**Comment by jarioksa** [comment, marked as answer]
`protest` calls `procrustes`  for the proper analysis, and `procrustes` uses `scores` with default settings. For constrained ordination methods `dbrda`, `capscale`, `rda` and `cca` the default setting is `scaling = "species"`. So it is not none `"none"`, but neither is it scaled by eigenvalues, and with `protest`setting  Why this is the default? Well, I really cannot remember certainly – `scores` with that choice (`scaling = 2`)  was released in May 2002. If I need to guess, it was because Canoco used that setting as default in that time. I guess you have found out that you can re-set the scaling with `protest(..., scaling = "sites")`. Arguably `scaling = "species"` is weird in `dbrda` that usually has no species scores to scale.

**Reply by grahamfrank** [reply]
Thanks, Jari. I hadn't seen that I could pass additional arguments to scores through the `protest` or `procrustes` function, which clarifies things for me. I agree that `scaling = "species"` doesn't make much sense for `dbrda` objects.
